src/theme.js

```diff
@@ -151,15 +231,26 @@ export default (overrides = {}) => {
     },
   };
 
+  const fonts = [
+    {
+      name: 'Tiempos',
+      url: '//cdn.heydoctor.com/fonts/TiemposHeadlineWeb-Semibold.woff2',
```

Comment:
I'm not sure if we need to include this in the theme - could you elaborate on how you seeing this used in the consuming apps?

Reply:
The goal here was to default as much as possible to HeyDoctor/GoodRx design, but I can understand leaving this out for the heydoctor theme file in each project.

Reply:
@kylealwyn It was just added to provide it as a default font. If we don't want to add it to the library we can add it individually to each app.